frontend/components/apps/AppCard.tsx

```diff
@@ -52,7 +89,37 @@ export const AppCard = (props: AppCardProps) => {
               {appEnvsData?.appEnvironments.length > 1 ? 'Environments' : 'Environment'}
             </span>
           </div>
+        {/* Component for Environments */}
+          {syncData?.sseEnabled && itemCount>0 && <div className="space-y-1">
+            <div className="flex items-center gap-2 text-2xl">
+               <FaProjectDiagram />
+              <span className="font-light">{itemCount}</span>
+            </div>
+            <span className="text-neutral-500 font-medium text-xs uppercase tracking-widest">
+              {itemCount > 1 ? 'Integrations' : 'Integration'}
+            </span>
+          </div>}
         </div>
+        <div className="flex justify-start items-center mt-[-1.6rem] pl-[20.5rem]">
+        {syncData?.sseEnabled &&
+              Object.entries(providerCounts).map(([provider, count]) => {
+                  if(count > 0 && iconPrint <= 5){
+                    iconPrint++;
+                  return (
+                    <div key={provider} className="flex-row flex items-center ml-2 ">
+                      <ProviderIcon providerId={provider} />
+                    </div>
+                  );
+                }
+                
+                return null;
+              })}
+              {syncData?.sseEnabled && noOfIcons > 5 &&
+                <div className="tfont-normal text-neutral-500 text-sm flex items-center mr-[-1.3rem] mt-[-0.5rem]">
+                  +n
+                </div>
+              }
```

Comment:
This approach uses a for loop inside Object.entries(which itself loops over the target array) within the render method. This creates a nested loop within the render loop, and could cause significant performance issues. Generally best to avoid for loop inside the render method.